src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -296,9 +312,22 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
       titleRef={titleRef}
       drawerRef={drawerRef}
       openCurrentEntitlementsListFromPanel={openCurrentEntitlementsListFromPanel}
+      deleteManifest={openDeleteConfirmation}
     />
   );
 
+  const actions = () => {
+    return [
+      {
+        title: 'Delete',
+        // eslint-disable-next-line
```

Comment:
I forgot about this.  The `rowData: any` was causing anger in the linter.

Reply:
ah. i think it just has the any as a warning now, at least for me, but if it's needed for now that's fine. We should probably tighten this up at some point in the future; I had left it a bit more flexible in large part because typing the react-query stuff was a bit tough and there was hardly any documentation on it.